android/WSL.README.md

```diff
@@ -0,0 +1,76 @@
+# Building BOINC Android client on Windows (WSL)
+
+## Goal
+
+Allow Windows users to use built-in WSL for setting up for BOINC Android development
+
+## Limitations
+
+* Windows 10 only
```

Comment:
- User must be running Windows 10 version 1607 (the Anniversary update) or above.
- WSL will only run on 64-bit versions of Windows 10. 32-bit versions are not supported.
- Also, if I remember correctly, only Windows 10 Pro or similar supports WSL

Reply:
Ok, actually Home can run WSL
https://ubuntu.com/wsl
https://devblogs.microsoft.com/commandline/wsl-2-post-build-faq/

Reply:
WSL2, not 1sr generation of WSL. As far as I know, WSL2 isn't released on stable channel yet

Reply:
Should I mention the further distinction of WSL1 and WSL2? I don't think it should be covered here as it's out of scope already. The point is WSL is available regardless of editions, only the build version and bitness matters

Reply:
Currently WSL can't be installed on Windows 10 Home. I suppose this month Microsoft will do a major update and WSL 2 will be officially released but I didn't see the exact date of this

Reply:
Windows 10 Home can run WSL1 (and soon WSL2)
https://en.wikipedia.org/wiki/Windows_10_editions#Comparison_chart
![VirtualBox_windows 10 home_06_05_2020_20_43_18](https://user-images.githubusercontent.com/12658589/81178277-8da0f580-8fda-11ea-878f-45001aafb4ec.png)